### DISCUSSION TITLE
Livewire.onError() doesn't contain error message

### DISCUSSION THREAD

**Discussion by reefki** [livewire/livewire#3351] [7. Bugs (archived)]
I'm implementing a new way to inform the user if there is something wrong. For example, I want to show a custom alert instead of showing the default Livewire modal.

I noticed that I can use `Livewire.onError()` to override the default Livewire error handling. 

```
Livewire.onError((message, statusCode) => {})
```

But it is only `statusCode` is available, while `message` is always `undefined`. Ideally it should also return the error message so that I can just use that message instead of writing hard-coded messages in my js files.

**Comment by lucasromanojf** [comment]
PR #4179

**Comment by lucasromanojf** [comment]
PR #4179 merged
